packages/ember-htmlbars/tests/helpers/get_test.js

```diff
@@ -1,495 +0,0 @@
-
-  run(function() {
-    view.set('context.colors.apple.gala', 'yellow-redish');
-  });
-
-  equal(view.$().text(), '[yellow-redish] [yellow-redish]', 'should return \'yellow-redish\' for {{get colors \'apple.gala\'}}');
-});
-
-QUnit.test('should be able to get an object value with a GetStream key', function() {
-  var context = {
-    colors: { apple: 'red', banana: 'yellow' },
-    key: 'key1',
-    possibleKeys: { key1: 'apple', key2: 'banana' }
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors (get possibleKeys key)}}] [{{if true (get colors (get possibleKeys key))}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\'');
-
-  run(function() {
-    view.set('context.key', 'key2');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'red\' for {{get colors key}} (key = \'banana\')');
-
-  run(function() {
-    view.set('context.colors', { apple: 'green', banana: 'purple' });
-  });
-
-  equal(view.$().text(), '[purple] [purple]', 'should return \'purple\'');
-
-  run(function() {
-    view.set('context.key', 'key1');
-  });
-
-  equal(view.$().text(), '[green] [green]', 'should return \'green\'');
-
-  run(function() {
-    view.set('context.colors.apple', 'red');
-  });
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\'');
-});
-
-QUnit.test('should be able to get an object value with a GetStream value and bound/dynamic key', function() {
-  var context = {
-    possibleValues: {
-      colors1: { apple: 'red', banana: 'yellow' },
-      colors2: { apple: 'green', banana: 'purple' }
-    },
-    objectKey: 'colors1',
-    key: 'apple'
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get (get possibleValues objectKey) key}}] [{{if true (get (get possibleValues objectKey) key)}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\'');
-
-  run(function() {
-    view.set('context.objectKey', 'colors2');
-  });
-
-  equal(view.$().text(), '[green] [green]', 'should return \'green\'');
-
-  run(function() {
-    view.set('context.objectKey', 'colors1');
-  });
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\'');
-
-  run(function() {
-    view.set('context.key', 'banana');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'yellow\'');
-
-  run(function() {
-    view.set('context.objectKey', 'colors2');
-  });
-
-  equal(view.$().text(), '[purple] [purple]', 'should return \'purple\'');
-
-  run(function() {
-    view.set('context.objectKey', 'colors1');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'yellow\'');
-});
-
-QUnit.test('should be able to get an object value with a GetStream value and GetStream key', function() {
```

Comment:
:heavy_check_mark: should be able to get an object value with a GetStream value and GetStream key